packages/menu-bar/test/a11y.test.js

```diff
@@ -0,0 +1,83 @@
+import { expect } from '@esm-bundle/chai';
+import { arrowDown, arrowRight, enter, fixtureSync, nextRender } from '@vaadin/testing-helpers';
+import '../src/vaadin-menu-bar.js';
+import { getDeepActiveElement } from '@vaadin/a11y-base/src/focus-utils.js';
+import { outsideClick } from './helpers.js';
+
+describe('a11y', () => {
+  describe('focus restoration', () => {
+    let menuBar, overlay, buttons;
+
+    beforeEach(async () => {
+      menuBar = fixtureSync(`<vaadin-menu-bar></vaadin-menu-bar>`);
+      menuBar.items = [
+        {
+          text: 'Item 0',
+          children: [
+            { text: 'Item 0/0' },
+            {
+              text: 'Item 0/1',
+              children: [{ text: 'Item 0/1/0' }],
+            },
+          ],
+        },
+      ];
+      overlay = menuBar._subMenu.$.overlay;
+      buttons = menuBar.querySelectorAll('vaadin-menu-bar-button');
+      buttons[0].focus();
+    });
+
+    it('should move focus to the sub-menu on open', async () => {
+      buttons[0].click();
+      await nextRender();
+      expect(getDeepActiveElement()).to.equal(overlay.$.overlay);
+    });
+
+    it('should restore focus on outside click', async () => {
+      // Open Item 0
+      arrowDown(getDeepActiveElement());
```

Comment:
**note:** The `sendKeys` API doesn't work for `menu-bar` for some non-obvious reason. It just doesn't do anything.